apps/e2ee-server/src/utils/RemoteApiProxyBase.ts

```
@@ -0,0 +1,129 @@
+  remoteApiType: 'e2eeServerApi',
+  context: IRoomManagerContext,
+) {
+  return async function callRemoteApiMethod(message: T) {
+    const { method, params = [] } = message;
+    // @ts-ignore
+    const module = message?.module as any;
+    if (!module) {
+      throw new Error('callRemoteApiMethod ERROR: module is required');
+    }
+    const moduleInstance: any = await moduleGetter(module);
+    if (moduleInstance && moduleInstance[method]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)